catalyst/data/tests/test_sampler.py

```diff
@@ -122,3 +126,62 @@ def test_balance_batch_sampler(
     """
     for labels, p, k in input_for_balance_batch_sampler:
         check_balance_batch_sampler_epoch(labels=labels, p=p, k=k)
+
+
+@pytest.fixture()
+def input_for_dynamic_balance_class_sampler() -> List[Tuple[list, float]]:
+    """
+    This function generates some valid inputs for DynamicBalanceClassSampler
+
+    Returns:
+        inputs in the folowing order: (labels, exp_l)
+    """
+    labels = [
+        # class imbalance
+        np.array([0] * 100 + [1] * 10 + [2] * 20),
+        # uniform class distribution
+        np.array([0] * 10 + [1] * 10 + [2] * 10),
+        # random class distribution
+        np.random.randint(0, 4, size=(200,)),
+    ]
+    exp_lambda = np.linspace(0.1, 0.95, 11)
+    input_cases = np.transpose(
+        [np.tile(labels, len(exp_lambda)), np.repeat(exp_lambda, len(labels))]
+    )
+    return input_cases
+
+
+def check_dynamic_balance_class_sampler(labels: List, exp_l: float) -> None:
+    """
+    Check DynamicBalanceClassSampler on certain inputs
+
+    Args:
+        labels: list of labels
+        exp_l: exponent figure for schedule
+    """
+    sampler = DynamicBalanceClassSampler(labels, exp_l)
+    n_labels = len(np.unique(labels))
+    labels_counter = Counter(labels)
+    min_class_key, min_class_size = labels_counter.most_common(n_labels)[-1]
+    current_d = {
+        key: value / min_class_size for key, value in Counter(labels).items()
+    }
+    for _ in range(10):  # noqa: WPS122
+        new_counter = Counter(labels[list(sampler.__iter__())])
+        new_d = {
+            key: value / min_class_size for key, value in new_counter.items()
+        }
+        for key, value in new_d.items():
+            assert value <= current_d[key]
+        assert new_d[min_class_key] == 1
+        current_d = new_d
+
+
+def test_dynamic_balance_class_sampler() -> None:
```

Comment:
please check how it should be done ;)
https://github.com/catalyst-team/catalyst/blob/master/catalyst/data/tests/test_sampler.py#L116#L124

Reply:
Ok )